app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Log in</h2>
+<h2 class="login-header">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "login-option" }) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] One of the requirements for this milestone is to ensure that the **_Log In and Sign Up_** pages are identical to the wireframe provided. Currently, they are not identical in terms of the design of the input boxes and buttons. Kindly redesign the **_Log In and Sign Up_** to look closely like the wireframe provided. 💻 👍 

#### Hint: Try working on the widths of the input boxes and the buttons. 😎 

### Your Design

![image](https://github.com/Megagig/Blog-App/assets/77106962/935755eb-ab59-4dd1-9834-3ec4fe68f3d6)

### Wireframe Provided

![image](https://github.com/Megagig/Blog-App/assets/77106962/36330738-3649-4d0a-9d53-097d6ff14d28)